uktrade/data-hub-frontend
# Feature/cls2 383 update adviser tab links to account management tab

## Description of change

The functionality to process adding, replacing and removing lead advisers on One List Tier D companies has moved from the Lead Adviser tab to the Account Management tab. It also updated links from the companies/[id]/advisers to companies/[id]/account-management from other Data Hub pages.

## Test instructions

When visiting an One List Tier D company's account management page you should be able to add, replace and remove a the Lead ITA. This functionality should not be present for Core team companies.

Lead ITA example: [One corp list](http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/account-management)
Core team example: [Venus solutions limited](http://localhost:3000/companies/0f5216e0-849f-11e6-ae22-56b6b6499611/account-management)

Other Data Hub pages should now link to companies/.../account-management instead of companies/.../advisers.

_What should I see?_

## Screenshots

### Before

<img width="977" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/699259/7e2fc189-39e7-41ba-bed0-e399bf8aec54">

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/08dbbf01-3c08-4368-b532-4aa1de000c15)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
